webapp/channels/src/utils/performance_telemetry/index.ts

```diff
@@ -13,6 +13,10 @@ export const enum Measure {
     ChannelSwitch = 'channel_switch',
     GlobalThreadsLoad = 'global_threads_load',
     PageLoad = 'page_load',
+    TTFB = 'TTFB',
+    TTLB = 'TTLB',
+    SplashScreen = 'splash_screen',
+    DomInteractive = 'dom_interactive',
```

Comment:
Nit: Could you sort the values of `Measure` alphabetically?

Reply:
We really need this to be done automatically via a linter.

Reply:
I'd tend to agree with that, but we do sometimes have cases where another order may make more sense. In this case, I'm only asking because I'd sorted the existing values previously. I think I only noticed because they're added to the middle of the list and not the end of it